wolftpm/tpm2_wrap.h

```diff
@@ -83,10 +83,25 @@ typedef struct WOLFTPM2_HMAC {
 } WOLFTPM2_HMAC;
 
 #ifdef WOLFTPM2_CERT_GEN
+
 typedef struct WOLFTPM2_CSR {
+#if defined(WOLFSSL_ASN_TEMPLATE) && defined(WOLFSSL_CUSTOM_OID) && \
```

Comment:
I prefer these defines to be outside the typedef although I realize it works this way too.